common/docker-entrypoint.sh

```diff
@@ -35,7 +35,7 @@ if [ -z ${DNS_RESOLVERS+x} ]; then
   resolvers=""
   for ip in $(grep nameserver /etc/resolv.conf | cut -d' ' -f2 | xargs)
   do
-    if echo "${ip}" | grep -q ':'; then
+    if echo "${ip}" | sed '/^[[:blank:]]*#/d;s/#.*//' | grep -q ':'; then
```

Comment:
Per the comment on the issue, did you look at https://github.com/nginxinc/docker-nginx/blob/master/entrypoint/15-local-resolvers.envsh for handling this?
It seems that in the case where `DNS_RESOLVERS` is not specified, we could potentially set it equal to the value of the env var exported by this script which I think should already have been run by this point?

Reply:
Oh, dang, I branched from the wrong commit. I'm going to redo this commit.